core/markets/storage/ask_stream.hpp

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_CORE_MARKETS_STORAGE__NETWORK_STORAGE_ASK_STREAM_HPP
+#define CPP_FILECOIN_CORE_MARKETS_STORAGE__NETWORK_STORAGE_ASK_STREAM_HPP
+
+#include "common/outcome.hpp"
+#include "markets/storage/ask_protocol.hpp"
+
+namespace fc::markets::storage {
+
+  /**
+   * StorageAskStream is a stream for reading/writing requests & responses on
+   * the Storage Ask protocol
+   */
+  class StorageAskStream {
```

Comment:
Remove this and use `CborStream` from #166.
It provides `read<T>`/`write<T>` to read/write cbor objects.
Also it exposes underlying `Stream`, so `close` can be called directly.

Reply:
👍